jenkins/continuous-integration.jenkins

```diff
@@ -0,0 +1,41 @@
+pipeline {
+  agent {
+    docker { image 'node:10-slim' }
+  }
+  environment {
+    CODECOV_TOKEN = '523afe7a-4620-466f-a26a-d914c18a7c0a'
+  }
+  stages {
+    stage('Build') {
+      steps {
+        sh 'yarn'
+      }
+    }
+    stage('Test') {
+      parallel {
+        stage('Lint') {
```

Comment:
Is there a reason for running the lint step _before_ the tests? I have found it useful to run lint checks after so that if the tests fail, I can fix them up before worrying about linting problems

Reply:
these stages are set to run in parallel so a failure in one branch doesn't stop the other branches from running. i thought it was best to collect as many results as possible in case of any failure.

Reply:
okies, I missed the `parallel` above it 😊